server.R

```diff
@@ -110,84 +134,43 @@ server <- function(input, output) {
                                measure_selection,
                                graph_type) {
     
-    # Measure names in drop down differ from those in data, use this to match them up
-    data <- if(measure_selection == "Total Population") {
-      filter(dataset, Measure == "Population.Index")
-      } else {
-        if(measure_selection == "Dependency Ratio") {
-          filter(dataset, Measure  == "Dependency.Ratio")
-          } else {
-            if(measure_selection == "Sex Ratio") {
-              filter(dataset, Measure  == "Sex.Ratio")
-            }
-          }
-      }
+    measure_title <- "Total Population Index"
     
-    # This will set the measure title for the graphs
-    # The graphs use population index data but the drop down is labelled total population so 
-    # use this to match them
-    measure_title <- if(measure_selection == "Total Population") {
-      "Total Population Index"
-    } else {
-      measure_selection
+    all_area_names <- unique(dataset$LongName)
+    
+    line_colours <- c("skyblue", "steelblue","dimgrey")
```

Comment:
Would change the order so that the small area (steelblue) is first in the legend. For this to work will need to change the order of the factor elsewhere in the code. I will update this.